src/asciidoc/docs/develop/addService.adoc

```diff
@@ -193,54 +189,126 @@ app.listen(process.env.PORT || 3333, null, async err => {
+  services: {
+    backend: process.env.REACT_APP_BACKEND_URL,
+    frontend: process.env.REACT_APP_FRONTEND_URL
+  },
+});
+....
+----
+
 == Letting Koji know about the backend service
 
-You have now added a backend project, but the Koji editor and project builder still don't know it exists.
-You must update the `develop.json` and `deploy.json` files to let Koji know about the new service.
-Both files are located in the `.koji` folder of the project folder.
+You have now added a backend project, but the Koji editor and app builder still don't know it exists.
+You must update the `koji.json` file to let Koji know about the new service.
+`koji.json` is located in the root directory.
 
-. Open the `.koji/project/develop.json` file and update it to look like the following code.
+. Open the `koji.json` file, and replace the existing content with the following code.
 +
 [source,json]
 ----
-include::../_includes/develop.json[tags=**,indent=0]
+{
+  "deploy": {
+    "frontend": {
+      "output": "frontend/build",
+      "type": "static",
+      "commands": [
+        "cd frontend",
+        "npm install",
+        "export NODE_ENV=production && npm run build"
+      ]
+    },
+    "backend": {
+      "output": "backend",
+      "type": "dynamic",
+      "commands": [
+        "cd backend",
+        "npm install",
+        "export NODE_ENV=production && npm run compile"
+      ]
+    }
+  },
+  "develop": {
+    "frontend": {
+      "path": "frontend",
+      "port": 8080,
+      "startCommand": "npm start"
+    },
+    "backend": {
+      "path": "backend",
+      "port": 3333,
+      "startCommand": "npm run start-dev"
+    }
+  },
+  "entitlements": {
+    "InstantRemixing": true,
+    "InstantRemixingNativeNavigation": true,
+    "FeedEvents": true
+  },
+  "remixData": {
+    "backgroundColor": "#282c34"
+  },
+  "@@initialTransform": {
+    "remixData": {}
+  }
+}
 ----
 +
-This file lets the Koji editor know which services to run, so you must add an entry for the `backend` service.
+This file lets the Koji editor know which services to run, so you must add entries for the `backend` service.
 . On the left side of the editor, go to *Advanced > Remote environment* and click *Force restart project*.
 +
 If everything went well, you should be able to reconnect to the editor and see that a `backend` terminal has been added at the bottom.
 +
 If you see `[koji] backend started`, your backend service is now running in your local editor environment.
-. To confirm that the `serviceMap` is now populated, open your preview tab and refresh it. You should now see a console log that looks like this example.
+
+== Testing the backend service
+
+To test that everything is working correctly, you can modify your frontend code to get a value from the backend service and display it automatically.
+
+. Go to the `frontend` folder, press CTL-C to stop the frontend server, and install the `styled-components` package, which the new code will use.
```

Comment:
We recently enabled some Asciidoc macros for keyboard instructions, buttons, and menus. As you update pages, can you start applying them to anything you see? Here’s the docs for the macros: https://docs.asciidoctor.org/asciidoc/latest/macros/keyboard-macro/, https://docs.asciidoctor.org/asciidoc/latest/macros/ui-macros/ Let me know if you have questions about how they work.

Also, I would make Install … a separate step.

```suggestion
. In the `frontend` terminal, press kbd:[CTRL+C] to stop the frontend server.
. Install the `styled-components` package, which the new code will use.
```